src/sezzle-checkout-installment-widget/sezzle-checkout-installment-widget.js

```diff
@@ -123,6 +123,23 @@ function renderInstallmentWidget(checkoutTotal, serviceRegion, currencySymbol){
 			},
 			'secondParagraph': 'Despu&#233;s de hacer clic en "Completar pedido" en este sitio, ser&#225; redirigido a Sezzle para completar su compra de forma segura.',
 			'infoIcon': 'M&#225;s informaci&#243;n sobre Sezzle'
+		},
+		'it': {
+		    'today': 'oggi',
+		    'days': 'giorni',
+		    'week': 'settimana',
+		    'month': 'mese',
+		    'installmentWidget': {
+			14: `4 rate senza interessi in 6 settimane`,
+			30: `4 rate in 3 mesi. Senza costi!`
+		    },
+		    'modalTitle': 'Come funziona',
+		    'firstParagraph': {
+			14: `Dividi l’intero ordine in 4 rate senza interessi in 6 settimane. Nessuna commissione se paghi in tempo e con impatto zero sul tuo credito.`,
```

Comment:
How concerned are you about charset compatibility? For all other translations, we have used Unicode for special characters for compatibility with ISO-8859-1. If you open the HTML file in your browser, you should see what I mean.

Reply:
You're right, I updated it in the next commit